### PR TITLE
Moves the dispatcher's local queue into memory

### PR DESCRIPTION
**Pull request by darrylmasson**
Rather than the database. The original idea was that this would help ensure continuity around dispatcher crashes/restarts, but I don't think this is a good idea/necessary anymore.

### CODE REVIEW

File: dispatcher/MongoConnect.py [hunk 1]
```diff
@@ -77,6 +76,10 @@ def __init__(self, config, daq_config, log, control_mc, runs_mc, hypervisor, tes
         # Which control keys do we look for?
         self.control_keys = config['ControlKeys'].split()
 
+        # a place to buffer commands temporarily
+        self.command_queue = []
+        self.q_mutex = threading.Mutex()
```
Comment (jorana) [inline dispatcher/MongoConnect.py:81]: I don't know this command, so might need some help here, also my python doesn't know it.

**Review by jorana — COMMENTED** [review]
Does this prevent other incoming commands and should we care?

File: dispatcher/MongoConnect.py (hunk 1 above)
Comment (darrylmasson) [reply dispatcher/MongoConnect.py:81]: In python it's apparently a `Lock`, but in every other language it's a mutex.

**Comment by darrylmasson** [comment]
This doesn't prevent any other commands, it's just the cleanest way of ensuring that commands are issued with the correct delay without holding up the logic. This PR moves this queue from the database into memory, because the issue the database was supposed to prevent never really showed up, and when it does it's better that we do run into it. Also when I first added this I was relying on some older, flawed ideas that don't work well, which is why longer-term buffering seemed necessary.

**Review by petergaemers — APPROVED** [review]

**Pull request merged**